### PR TITLE
issue #152: New depandabot.yml

### PR DESCRIPTION
**Pull request by RickDeb2004**
Dependabot will now automatically create pull requests to update outdated dependencies.
Signed-off-by: Debanjan Mukherjee <debanjanrick04@gmail.com>

### CODE REVIEW

**Review by TerryHowe — CHANGES_REQUESTED** [review]
I think this needs to be up one dir like https://github.com/oras-project/oras/blob/main/.github/dependabot.yml and have the copyright I guess.

Also, you need to sign the DCO.

Thanks!

**Comment by RickDeb2004** [comment]
> I think this needs to be up one dir like https://github.com/oras-project/oras/blob/main/.github/dependabot.yml and have the copyright I guess.
> 
> Also, you need to sign the DCO.
> 
> Thanks!

Sir ,I have added the signed off message still DCO check is not running ,pls check this issue
and I think this needs to be up one dir like https://github.com/oras-project/oras/blob/main/.github/dependabot.yml I can't understand this one ,pls elaborate it

File: .github/ISSUE_TEMPLATE/depandabot.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright The ORAS Authors.
```
Comment (TerryHowe) [inline .github/ISSUE_TEMPLATE/depandabot.yml:1]: I'm prettu sure this file should be in the `.github` directory

Comment (TerryHowe) [reply .github/ISSUE_TEMPLATE/depandabot.yml:1]: Your first commit is not signed off and I think that is causing the DCO problem. You might want to squash the 3 commits into one like https://stackoverflow.com/questions/5189560/how-do-i-squash-my-last-n-commits-together and `git commit -signoff` on the one commit. After that force push the changes

**Review by TerryHowe — CHANGES_REQUESTED** [review]

**Comment by shizhMSFT** [comment]
Closed by #162

**Pull request closed without merging**